Jenkinsfile

```diff
@@ -95,19 +95,23 @@ pipeline {
             sh './bin/helm-dependency-update-in-docker'
           }
         }
-        stage('Openshift E2E Workflow Tests') {
-          steps {
-            sh 'cd bin/test-workflow && summon --environment openshift -D ENV=ci -D VER=current ./start --platform oc'
-          }
-        }
-        stage('Run E2E Tests') {
+        stage('Test app with') {
           parallel {
-            stage('Enterprise and test app deployed to GKE') {
+            stage('Enterprise in GKE') {
               steps {
                 sh 'cd bin/test-workflow && summon --environment gke ./start --enterprise --platform gke'
               }
             }
-            stage('Enterprise deployed locally, test app deployed to GKE') {
+            stage('OSS in OpenShift') {
+              steps {
+                sh 'cd bin/test-workflow && summon --environment openshift -D ENV=ci -D VER=current ./start --platform openshift'
+              }
+            }
+          }
+        }
+        stage('Enterprise in Jenkins') {
+          stages {
+            stage('Test app in GKE') {
```

Comment:
Could we run these stages in parallel?
If not, I think they probably compete to deploy Enterprise to the same port. Wondering if we could deploy one Conjur instance, and have parallel stages authenticate with the same Conjur instance. Not sure if that would even work, just a thought, and definitely out of scope.

Reply:
"These stages" referring to `Test app in GKE` and `Test app in Openshift`

Reply:
Two Enterprise installs in the same Jenkins agent clobber each other when run in parallel. A way around this would be to do the Enterprise installs in containers with container listening ports being forwarded to different Jenkins agent listening ports - and then configuring the authenticators to talk to Enterprise on these ports - but I think its more trouble than its worth.

Reply:
A single Enterprise instance with the policy loading involved to support multiple authenticators and test apps sounds pretty difficult to troubleshoot and not worth it for what we're trying to achieve here IMO - I think we want to keep these E2E flows simple and easy to debug even at the expense of some parallelism

Reply:
Good point, and you're right - more trouble than it's worth.